src/routes/entry-point-route-object.types.ts

```diff
@@ -16,15 +16,15 @@ export interface EntryPointIndexRouteObject
     IndexRouteObject,
     "loader" | "action" | "element" | "Component" | "lazy"
   > {
-  entryPoint: BadEntryPointType;
+  entryPoint: LazyLoadable<BadEntryPointType>;
```

Comment:
How about instead of a new `LazyLoadable` type we use `JSResource` here as well. I think that'll help us avoid issues if two or more routes share an entry point.

```suggestion
  entryPoint: BadEntryPointType | JSResourceReference<BadEntryPointType>;
```

Reply:
Sounds reasonable! I'll do that soon.

Reply:
https://github.com/loop-payments/react-router-relay/pull/1/commits/56080550d7cf947bf48d49bc236d264f5faf9218